modules/network_encoding.go

```diff
@@ -339,10 +339,10 @@ func (d *DNAT) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON implements encoding/json.Unmarshaler
 func (e *ExitPoint) UnmarshalJSON(b []byte) error {
 	tmp := struct {
-		Ipv4Conf  *Ipv4Conf `json:"ipv4_conf"`
-		Ipv4Dnat  []*DNAT   `json:"ipv4_dnat"`
-		Ipv6Conf  *Ipv6Conf `json:"ipv6_conf"`
-		Ipv6Allow []net.IP  `json:"ipv6_allow"`
+		Ipv4Conf  *Ipv4Conf    `json:"ipv4_conf"`
+		Ipv4Dnat  []*DNAT      `json:"ipv4_dnat"`
+		Ipv6Conf  *Ipv6Conf    `json:"ipv6_conf"`
+		Ipv6Allow []*Ipv6Allow `json:"ipv6_allow"`
```

Comment:
Any reason this was defined as list of pointers instead of list of struct?
```suggestion
		Ipv6Allow []Ipv6Allow `json:"ipv6_allow"`
```